pod-operation/src/demo.rs

```diff
@@ -1,46 +1,62 @@
 use tracing::info;
 
-use crate::components::lim_temperature::LimTemperature;
+use crate::components::lim_current::LimCurrent; // Changed from hall_effect to lim_current
```

Comment:
Chore [repeating]: descriptions of changes as comments do not have lasting value. If you would like to state what's changed in a commit, you can include a longer commit message body.